libs/langchain/langchain/vectorstores/redis/filters.py

```diff
@@ -167,12 +181,10 @@ def _formatted_tag_value(self) -> str:
         return "|".join([self.escaper.escape(tag) for tag in self._value])
 
     def __str__(self) -> str:
+        """Return the query syntax for a RedisTag filter expression."""
         if not self._value:
-            raise ValueError(
-                f"Operator must be used before calling __str__. Operators are "
-                f"{self.OPERATORS.values()}"
-            )
-        """Return the Redis Query syntax for a RedisTag filter expression"""
+            return "*"
```

Comment:
Is this a commonly understood empty expression in redis? A little code comment here would be helpful

Reply:
Good callout. I will add a comment clarifying the implementation!